app/classifier.py

```diff
@@ -1,18 +1,56 @@
+from operator import itemgetter
+import logging
+import string
+import time
+import os
+
+import numpy as np
+import tensorflow as tf
+
 from app import embedder
+from app import model_fetcher
+
+
+def fetchVocab(vocab: str):
+    with open(vocab, 'r') as f:
+        return f.readlines()
 
 
 class Classifier:
 
-    def __init__(self):
-        self.embedder = embedder.Embedder()
+    def __init__(self, bert, classifier, vocab: str, prefetch=False):
```

Comment:
doc doc doc

(especially since some of these are handed directly to TF, their structure should be clear)

Reply:
Done.